packages/node/src/lib/execution-details/execution-details.ts

```diff
@@ -0,0 +1,8 @@
+import { IExecutionDetails } from './execution-details.interface';
+import { WithHttp } from '../novu.interface';
+
+export class ExecutionDetails extends WithHttp implements IExecutionDetails {
+  async get() {
+    return await this.http.get(`/execution-details`);
```

Comment:
You missed params notificationId and subscriberId for this https://api.novu.co/api#/Execution%20Details/ExecutionDetailsController_getExecutionDetailsForNotification

Reply:
good spot, thanks.
fixed!